cgmanifest.json

```diff
@@ -7976,6 +7976,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "k3s",
+          "version": "1.25.5",
```

Comment:
We should drop the entry for `1.25.0` (line 7970-7975) since we no longer ship that.

Reply:
Done